### PR TITLE
Add EmptyTryBlock rule

### PR DESCRIPTION
**Pull request by schalkms**

### CODE REVIEW

File: detekt-rules/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/empty/EmptyTryBlock.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package io.gitlab.arturbosch.detekt.rules.empty
+
+import io.gitlab.arturbosch.detekt.api.Config
+import org.jetbrains.kotlin.psi.KtTryExpression
+
+/**
+ * Reports empty `try` blocks. Empty blocks of code serve no purpose and should be removed.
+ *
+ * @active since v1.5.1
```
Comment (3flex) [inline detekt-rules/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/empty/EmptyTryBlock.kt]: Update with next release number?

Comment (schalkms) [reply detekt-rules/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/empty/EmptyTryBlock.kt]: @arturbosch do we go straight to 1.6 or is another patch version coming?

Comment (BraisGabin) [reply detekt-rules/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/empty/EmptyTryBlock.kt]: We should not activate any rule in a patch version.

Comment (schalkms) [reply detekt-rules/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/empty/EmptyTryBlock.kt]: That’s true.

**Pull request merged**